bundle/tests/scorecard/kuttl/storage/03-runtime-storage-class-name.yaml

```diff
@@ -0,0 +1,15 @@
+# With some storage.className configured, the operator
+# should create a PVC with the configured storageClassName
+apiVersion: rc.app.stacks/v1beta2
+kind: RuntimeComponent
+metadata:
+  name: storage-rc
+spec:
+  # Add fields here
+  applicationImage: 'k8s.gcr.io/pause:2.0'
```

Comment:
Please remove the quotes around the image as it'll cause problem with the minikube scenario (the changes Melissa is making). The replicas: 1 can also be removed since that's the default.